lib/generator.js

```diff
@@ -990,6 +990,8 @@
                 outputRect: settings.outputRect,
                 scaleX:     settings.scaleX || 1,
                 scaleY:     settings.scaleY || 1,
+                targetWidth: settings.targetWidth,
+                targetHeight: settings.targetHeight,
```

Comment:
If you supply only one of these target values, does PS do a reasonable thing?  See also: these setting need to be in JSDoc somewhere